comp/core/tagger/origindetection/origindetection.go

```diff
@@ -39,25 +39,25 @@ const (
 
 // OriginInfo contains the Origin Detection information.
 type OriginInfo struct {
-	LocalData     LocalData     // Local Data is the local data list.
+	LocalData     LocalData     // LocalData is the local data list.
 	ExternalData  ExternalData  // ExternalData is the external data list.
 	Cardinality   string        // Cardinality is the cardinality of the resolved origin.
 	ProductOrigin ProductOrigin // ProductOrigin is the product that sent the origin information.
 }
 
 // LocalData that is generated by the client and sent to the Agent.
 type LocalData struct {
-	ContainerIDFromSocket string
-	ContainerID           string
-	Inode                 string
-	PodUID                string
+	ProcessID   uint32 // ProcessID of the container process on the host.
+	ContainerID string // ContainerID sent from the client.
+	Inode       uint64 // Inode is the Cgroup inode of the container.
+	PodUID      string // PodUID of the pod sent from the client.
 }
 
 // ExternalData generated by the Admission Controller and sent to the Agent.
 type ExternalData struct {
-	Init          bool
-	ContainerName string
-	PodUID        string
+	Init          bool   // Init is true if the container is an init container.
+	ContainerName string // ContainerName is the name of the container as seen by the Admission Controller.
```

Comment:
IMO, we should extend the description to something more generic

```suggestion
ContainerName string // ContainerName is the name of the container in the Kubernetes Pod spec in case of Kubernetes, or the container name set by the container runtime for the other cases.
```